risc0/build_kernel/src/lib.rs

```diff
@@ -136,8 +136,7 @@ impl KernelBuild {
 
     fn compile_cuda(&mut self, output: &str) {
         println!("cargo:rerun-if-env-changed=RISC0_CUDA_OPT");
-        println!("cargo:rerun-if-env-changed=NVCC_PREPEND_FLAGS");
-        println!("cargo:rerun-if-env-changed=NVCC_APPEND_FLAGS");
+        println!("cargo:rerun-if-env-changed=RISC0_NVCC_FLAGS");
```

Comment:
@mothran @rlukata Note: this was changed so that by default, `-arch=native` is used. In order to build for a specific architecture, use `RISC0_NVCC_FLAGS` instead of `NVCC_PREPEND_FLAGS` or `NVCC_APPEND_FLAGS`.